source/user_guide/basic_capabilities/feature_attributes.rst

```diff
@@ -91,26 +120,6 @@ How do I specify dates or times?
       "end_time": "%I:%M:%S %p",
     }
 
```

Comment:
We should probably retain the docs for Partial Features until it is actually removed. Do, however, update this section to make it clear the functionality is deprecated (as of release: ...) and will be removed in a future release.